docs/design_docs/011-historacle-pricing.md

```diff
@@ -34,21 +35,31 @@ Historacle pricing will provide an API for the leverage module to tell when pric
 
 ## Specification
 
+These values are stored in state in order to avoid the `x/leverage` module from having to calculate them while making decisions around allowable positions for users to take.
+
+### Epochs
+
 We define three epoch periods, during which additional computation will be performed:
-- `Stamp Period`: duration during which the `x/oracle` module will now "stamp" the set of exchange rates in the state machine until a `Pruning Period` has passed (30 days).
-- `Pruning Period`: duration after which the `x/oracle` module will begin to prune expired historic prices.
-- `Median Period`: will determine how often the Median and the `Standard Deviation around the Median` are calculated, which will also be stored in the state machine.
 
-These values are stored in state in order to avoid the `x/leverage` module from having to calculate them while making decisions around allowable positions for users to take.
+- `Historic Stamp Period`: duration during which the `x/oracle` module will now "stamp" the set of exchange rates in the state machine until a `Pruning Period` has passed (30 days).
```

Comment:
This sounds like we're keeping the once-per-5-block stamps for 30 days. I thought we were clearing every 6hrs

Reply:
Clarified this bit.

Tldr; we'll stamp as often as we like (maybe every 100 blocks?) until a `Max Historic Prices` is met. So, you can stamp every 100 blocks, and every stamp will be valid for 6 hours.